tests/ImageSharp.Tests/project.json

```diff
@@ -20,9 +20,24 @@
     }
```

Comment:
Happy to be merciful if it helps productivity. Let's all try to clean up as we update tests and make sure we don't introduce new formatting issues and we'll be fine 😄.